challenges/2-tabuada/js/xandowski/index.js

```diff
@@ -0,0 +1,12 @@
+const tabuada = (n) => {
+  for(let i = 1; i <= n; i++){
+    console.log(`A tabuada do número ${i} é: `)
+    for(let j = 1; j < 11; j++) {
+      console.log(`${i} x ${j} = ${i * j}`);
+    }
+    console.log("===========================")
+  }
+  return 'He4rtoberfest!!';
+}
+
+console.log(tabuada(2));
```

Comment:
Salve primo, tudo tranquilo? 
Tenho uma dúvida: dentro da sua função já está sendo usado `console.log`, certo? Por quê quando você invoca a função ela está dentro de um console log? Creio que seja indiferente ter esse código extra na sua aplicação.

```suggestion
tabuada(2);
```

Reply:
tem razão não precisa dele ja arrumei vlw primo.